Modules/Server/Sources/PocketCastsServer/Public/Search/PodcastSearchTask.swift

```diff
@@ -1,9 +1,29 @@
 import Foundation
+import PocketCastsDataModel
 
-public struct PodcastSearchResult: Codable, Hashable {
+public struct PodcastFolderSearchResult: Codable, Hashable {
```

Comment:
I don't like the idea of this representing a Podcast OR a Folder.

However, I tried encapsulating similar properties into a protocol and having both separated but didn't manage to succeed. Any ideas are appreciated.